### PR TITLE
Create FDB_installation_zh.md

### PR DESCRIPTION
**Pull request by LorreinMa**
I hereby agree to the terms of the CLA available at: [placeholder]

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

...Add Chinses doc FDB_installation

Detailed description / Documentation draft:

...

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ByConity Documentation](https://github.com/ByConity/ByConity/blob/master/CONTRIBUTING.md) guide first.

### CODE REVIEW

**Comment by LorreinMa** [comment]
@kevinthfang pls review

File: docker/executable_wrapper/FDB_installation_zh.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+# Foundation DB 安装指南
+在本指南中，我将在3台使用Debian OS的物理机上设置一个FoundationDB集群。 可以参考[Getting Started on Linux](https://apple.github.io/foundationdb/getting-started-linux.html) and [Building a Cluster](https://apple.github.io/foundationdb/building-cluster.html)两个官方指南。
+
+首先，我们需要在[此处](https://github.com/apple/foundationdb/releases/)下载并安装的二进制文件。 我们需要下载**服务器**，**监视器**和**CLI**二进制文件，以及那些coressponding **sha256**校验和文件，本指南选择撰写时的最新版本**7.1.25**版。 接着创建一个文件夹并使用以下命令下载：
```
Comment (kevinthfang) [inline docker/executable_wrapper/FDB_installation_zh.md:4]: Use english names in such cases

File: docker/executable_wrapper/FDB_installation_zh.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+```Plaintext
+$ sha256sum --binary fdbserver.x86_64
+73b70a75464e64fd0a01a7536e110e31c3e6ce793d425aecfc40f0be9f0652b7 *fdbserver.x86_64
+
+$ cat fdbserver.x86_64.sha256
+73b70a75464e64fd0a01a7536e110e31c3e6ce793d425aecfc40f0be9f0652b7  fdbserver.x86_64
+```
+
+假设您下载它们并存储在目录/`/root/user_xyz/foundationdb/bin`中。 接下来删除那些sha256 checksum文件，因为我们不再需要它们。我们将重命名可执行文件（executation file）以删除Tailting`x86_64` 并授予它们可执行的许可（executable permission）。
```
Comment (kevinthfang) [inline docker/executable_wrapper/FDB_installation_zh.md:27]: 可以精简一点，让翻译痕迹不要那么重，比如
假设它们被存储在目录/`/root/user_xyz/foundationdb/bin`中。 接下来可以删除那些sha256 checksum文件，因为不再需要使用。我们可以重命名可执行文件并授予它们可执行的权限。
下同

File: docker/executable_wrapper/FDB_installation_zh.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+clusterdsc:wwxVEcyLvSiO3BGKxjIw7Sg5d1UTX5ad@10.21.154.30:4500,10.21.170.163:4500,10.149.54.214:4500
+```
+
+将此文件复制到其他2台机器并替换旧文件然后重新启动fdb服务
+
+```Plaintext
+systemctl restart fdb.service
+```
+
+然后回到第一台机器，执行这条命令，将冗余模式改为 `triple`
```
Comment (kevinthfang) [inline docker/executable_wrapper/FDB_installation_zh.md:170]: 删除这个“将冗余模式改为 `triple`”

**Review by kevinthfang — CHANGES_REQUESTED** [review]

**Review by kevinthfang — APPROVED** [review]

**Pull request merged**